### PR TITLE
docs: link to specific sponsors

### PR DESCRIPTION
**Pull request by dwelle**

### CODE REVIEW

**Comment by dwelle** [comment]
I guess no rounded corners :(

**Comment by lipis** [comment]
How hard is to use the API to get the top sponsors?

File: README.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Go to [excalidraw.com](https://excalidraw.com) to start sketching.
 
 Read the latest news and updates on our [blog](https://blog.excalidraw.com). A good start is to see all the updates of [One Year of Excalidraw](https://blog.excalidraw.com/one-year-of-excalidraw/).
 
-## We accept donations
+## Supporting Excalidraw
```
Comment (lipis) [inline README.md:23]: I like the more direct approach... /cc @vjeux?

Comment (ad1992) [reply README.md:23]: I feel this is a better way of communicating to viewers that we need their support :)

**Review by ad1992 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lipis) [reply README.md:23]: @ad1992 which one do you mean/prefer?

Comment (ad1992) [reply README.md:23]: The new one `Supporting Excalidraw`

**Pull request merged**